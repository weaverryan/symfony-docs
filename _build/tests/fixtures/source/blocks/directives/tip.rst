
.. tip::

    This is a little tip about something! We an also talk about specific
