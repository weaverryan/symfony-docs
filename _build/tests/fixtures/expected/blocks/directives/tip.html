<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>
<div class="admonition-tip admonition-wrapper">
    <div class="tip"></div>
    <div class="admonition admonition-tip">
        <p class="admonition-title">Tip</p>
        <p>This is a little tip about something! We an also talk about specific</p>
    </div>
</div>
</body>
</html>